addon/components/o-s-s/empty-state.ts

```diff
@@ -0,0 +1,21 @@
+import type { htmlSafe } from '@ember/template';
+import Component from '@glimmer/component';
+
+interface OSSEmptyStateComponentSignature {
+  badgeIcon?: string;
+  title: ReturnType<typeof htmlSafe>;
+  subtitle: ReturnType<typeof htmlSafe>;
+  size?: 'sm' | 'md';
+}
+
+const ALLOWED_SIZE = ['sm', 'md'];
```

Comment:
```suggestion
const ALLOWED_SIZES = ['sm', 'md'];
```
My bad I forget the "s" at the end, don't forget the type also